### PR TITLE
add create_time to information_schema.tables (issue #1909)

### PR DESCRIPTION
**Pull request by yangzhg**
Add create_time to information_schema.tables (issue #1909)

- be/src/exec/schema_scanner/schema_tables_scanner.cpp
-  fe/src/main/java/org/apache/doris/catalog/Table.java
- fe/src/main/java/org/apache/doris/common/FeConstants.java
- fe/src/main/java/org/apache/doris/common/FeMetaVersion.java
- fe/src/main/java/org/apache/doris/service/FrontendServiceImpl.java
- fe/src/test/java/org/apache/doris/catalog/TableTest.java
- gensrc/thrift/FrontendService.thrift

### CODE REVIEW

**Comment by imay** [comment]
@yangzhg 
Hi, can you split this PR into two PRs? one for adding create_time and one for compile error?

**Comment by yangzhg** [comment]
> @yangzhg
> Hi, can you split this PR into two PRs? one for adding create_time and one for compile error?

removed compile error code

File: be/src/exec/schema_scanner/schema_tables_scanner.cpp [hunk 1]
```diff
@@ -176,15 +176,30 @@ Status SchemaTablesScanner::fill_one_row(Tuple *tuple, MemPool *pool) {
...
     }
     // check_time
     {
-        tuple->set_null(_tuple_desc->slots()[16]->null_indicator_offset());
+        void *slot = tuple->get_slot(_tuple_desc->slots()[16]->tuple_offset());
+        DateTimeValue *time_slot = reinterpret_cast<DateTimeValue*>(slot);
+        int64_t check_time = tbl_status.last_check_time;
+        if (check_time <= 0) {
+            tuple->set_null(_tuple_desc->slots()[16]->null_indicator_offset());
+        } else {
```
Comment (imay) [inline be/src/exec/schema_scanner/schema_tables_scanner.cpp]: set_not_null

File: be/src/exec/schema_scanner/schema_tables_scanner.cpp [hunk 2]
```diff
@@ -176,15 +176,30 @@ Status SchemaTablesScanner::fill_one_row(Tuple *tuple, MemPool *pool) {
     }
     // creation_time
     {
-        tuple->set_null(_tuple_desc->slots()[14]->null_indicator_offset());
+        void *slot = tuple->get_slot(_tuple_desc->slots()[14]->tuple_offset());
+        DateTimeValue *time_slot = reinterpret_cast<DateTimeValue*>(slot);
+        int64_t create_time = tbl_status.create_time;
+        if (create_time <= 0) {
+            tuple->set_null(_tuple_desc->slots()[14]->null_indicator_offset());
+        } else {
```
Comment (imay) [inline be/src/exec/schema_scanner/schema_tables_scanner.cpp]: set_not_null first

**Review by imay — CHANGES_REQUESTED** [review]

File: be/src/exec/schema_scanner/schema_tables_scanner.cpp (hunk 2 above)
Comment (yangzhg) [reply be/src/exec/schema_scanner/schema_tables_scanner.cpp]: done

File: be/src/exec/schema_scanner/schema_tables_scanner.cpp (hunk 1 above)
Comment (yangzhg) [reply be/src/exec/schema_scanner/schema_tables_scanner.cpp]: done

File: be/src/exec/schema_scanner/schema_tables_scanner.cpp (hunk 2 above)
Comment (yangzhg) [reply be/src/exec/schema_scanner/schema_tables_scanner.cpp]: Done

**Review by imay — APPROVED** [review]
LGTM

**Review by kangkaisen — APPROVED** [review]
+1. LGTM.

**Pull request merged**